README.md

```diff
@@ -6,7 +6,7 @@
 
 To install this script in KolMafia, simply use the following command in the gCLI:
 
-`svn checkout https://github.com/RESPRiT/bountiful/branches/release`
+`git  checkout https://github.com/loathers/bountiful.git`
```

Comment:
i believe we can now (starting with r26691) use `git checkout loathers/bountiful main` but i am not sure, we should test this. and i am open to the idea that we should (for backwards compatibility reasons) just use the URL

Reply:
Good call, I like the simpler approach. Believe we don't need to include `main` at the end either, it is the default. I'll test it out

Reply:
Yep `git checkout loathers/bountiful` works great